develop-docs/sdk/javascript/browser-tracing.mdx

```diff
@@ -0,0 +1,151 @@
+pageload span as early as possible. However, this still means that the span is only started once the SDK code is shipped to the browser. In reality though, the actual page load
+starts with the browser making the request for the URL to the server. To account for this, we retroactively backdate the start time of the pageload span to the browser request
+start time. We are able to backdate the start time because the browser provides this timing information.
+
+The pageload span is **earliest** ended, when the browser emits `'interactive'` or `'complete'` in `WINDOW.document.readyState`. It might be very well prolonged by our instrumentation adding additional child spans to the pageload root span.
+
+![Pageload Span](pageload-span.png)
+
+A pageload span:
+
+- Always has the `sentry.op: 'pageload'` attribute
+- Always is a root span
+- [Continues a trace](#trace-continuation) picked up from `<meta>` tags in the initial HTML response or, if no tags are detected, starts a new trace.
+- Contains various spans with specific ops that we retroactively create from browser Performance API entries (see image for example spans):
+  1. `browser` - information about the pageload request lifecycle (entry type `navigation`).
+  2. `resource.*` - resource timing information containing the duration of how long images, CSS, JS files etc. took to load.
+  3. `measure` - custom time measurements. Some of them are set by our SDK, some by SDK users or 3rd parties, some by the browser.
+  4. `mark` - custom time markers. Some of them are set by our SDK, some SDK users or 3rd parties, some by the browser.
+  5. `paint` - paint timing information.
+  6. `ui.long-animation-frame` - a span showing the duration of a long animation frame event detected by the browser.
+- Contains [web vital](#web-vitals) as measurements.
+- May contain child spans created from other instrumentations (e.g. `http.client`) spans or manually created spans.
+
+Since the pageload span is automatically started at SDK initialization, **there's only one pageload span** started during the SDK lifecycle.
+
+### Navigation Spans
+
+Navigation spans are started when a "soft navigation" occurs.
+A "soft navigation" means that the URL or history state of the page changes **without** a full page reload.
+By default, in `@sentry/browser` the SDK listens to the browser's `History` API to detect such navigations.
+Framework-specific SDKs and router instrumentations might use other mechanisms to detect them.
+
+Like a pageload span, navigation spans are idle spans and end themselves after a certain time of inactivity or after a maximum duration.
+
+In contrast to pageload spans, navigation spans do not contain web vital measurements. Read more about the reason [here](#web-vitals).
+
+### Router Instrumentations and Route Parameterization
+
+Some frameworks like React, Angular or Vue have their own in-browser routing solutions (e.g. React Router).
```

Comment:
```suggestion
Some frameworks like React, Angular or Vue have their own in-browser routing solutions (e.g. Angular Router).
```
React Router is maybe not the best example here, as it is not "in" the framework. One could also use the tanstack router.